components/hab/src/command/pkg/export/helm.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2017 Chef Software Inc. and/or applicable contributors
```

Comment:
2018! 😄

Reply:
The file was created in 2017 as you can see in the author date on the commit. :)